proto/api/v1/api.proto

```diff
@@ -669,6 +669,20 @@ service ProcessorService {
     };
   };
 
+  rpc InspectProcessorIn(InspectProcessorInRequest) returns (stream InspectProcessorInResponse) {
+    option (google.api.http) = {
+      get: "/v1/processors/{id}/inspect-in"
+      response_body: "record"
+    };
+  }
+
+  rpc InspectProcessorOut(InspectProcessorOutRequest) returns (stream InspectProcessorOutResponse) {
+    option (google.api.http) = {
+      get: "/v1/processors/{id}/inspect-out"
+      response_body: "record"
+    };
+  }
```

Comment:
We have 3 options to specify which records we want to inspect:
1. query params, e.g. `/v1/processors/{id}/inspect?type=in`
2. path params, e.g. `/v1/processors/{id}/inspect/{type}`
3. different endpoints (as implemented)

Query params would mean a sane default exists, which doesn't apply in our case. 

Path params vs different endpoints: Usage-wise, pretty similar. Different endpoints are less work (no type validation). It also makes it clear that only these two types are supported (everything else is a 404 error code). Also, path params indicate there are two `inspect` resources (with IDs `in` and `out`), but `inspect` here is meant as a verb (to start inspecting).

Reply:
I'm not sure which approach is better, none of them strike me as particularly nice tbh, but I don't have a better proposal. I slightly lean towards option 2, although I have nothing to back up my preference except it "looks nicer" 😅

Another thing to think about is changing in/out to incoming/outgoing, although I don't feel strongly about that either. I don't mind if you leave everything as it is now, just throwing out more ideas as food for thought 😉

Reply:
I do agree that option 2 looks nicer, but I think that it makes it less REST-like. It would indicate there's an `inspect` resource type and two different resources with IDs `in` and `out` (or `incoming` and `outgoing`). But again, I find it definitely prettier.:)

`incoming` and `outgoing` also sound good to me. I lean a bit more towards `in` and `out` because it's shorter, but still clear what it's about.